RCAIDE/Library/Components/Energy/Networks.py

```diff
@@ -0,0 +1,85 @@
+# RCAIDE/Library/Components/Energy/Network.py
+# (c) Copyright 2025 Aerospace Research Community LLC
+#
+# Created: Apr 2025, RCAIDE Team
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  IMPORT
+# ----------------------------------------------------------------------------------------------------------------------
+
+import chex
+from dataclasses import field, make_dataclass
+
+# package imports
+import numpy as np
+
+# RCAIDE imports
+import RCAIDE.Framework as rcf
+import RCAIDE.Library as rcl
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  Network
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+@chex.dataclass(kw_only=True)
+class EnergyLine(rcl.Component):
+
+    propulsors:     rcl.Component = field(default_factory=lambda: rcl.Component(tag='Propulsors'))
+    converters:     rcl.Component = field(default_factory=lambda: rcl.Component(tag='Converters'))
+    stores:         rcl.Component = field(default_factory=lambda: rcl.Component(tag='Stores'))
+
+    def add_subcomponent(
+            self,
+            subcomponent: rcl.Component,
+            sum_mass=False,
+            sum_center_of_gravity=False,
+            sum_moments_of_inertia=False
+    ):
+
+        if isinstance(subcomponent, rcl.Components.Energy.Propulsors.Propulsor):
+            self.propulsors.add_subcomponent(subcomponent, sum_mass, sum_center_of_gravity, sum_moments_of_inertia)
+        elif isinstance(subcomponent, rcl.Components.Energy.Converters.EnergyConverter):
+            self.converters.add_subcomponent(subcomponent, sum_mass, sum_center_of_gravity, sum_moments_of_inertia)
+        elif isinstance(subcomponent, rcl.Components.Energy.Stores.EnergyStore):
+            self.stores.add_subcomponent(subcomponent, sum_mass, sum_center_of_gravity, sum_moments_of_inertia)
+        else:
+            super(EnergyLine, self).add_subcomponent(subcomponent, sum_mass, sum_center_of_gravity, sum_moments_of_inertia)
+
+    def __post_init__(self):
+        self.add_subcomponent(self.propulsors)
```

Comment:
In the future these need to updated. Propulsors, converters, and stores are insufficient to define a network.